recipes/psi4/conda_build_config.yaml

```diff
@@ -0,0 +1,21 @@
+channel_sources:
+  - conda-forge/label/libint_dev,conda-forge
+channel_targets:
+  - conda-forge psi4_dev
```

Comment:
> The primary concern I anticipate is that this isn't packaging an upstream release, only an upstream branch. This is because the psi4 conda package and developer build instructions have always been based on -c defaults -c psi4, but there's one big and accumulating little changes to prepare for c-f deps, and I'd like to preserve the usability of the existing instructions until the whole software stack is ready on c-f. So if this recipe is merged, I expect that the first release from a c-f feedstock will have an ugly version (e.g., 1.8a2). I'd be glad to have it on a dev branch and psi4_dev channel if that can be done at feedstock-creation-time. A c-f compatible psi4 v1.8 release is due for late April/early May.

Using a dev channel is the correct way to address this pre-release problem. Good job.

Reply:
Great, thanks. I appreciate these dev channels where one can polish the code and recipes.